languages/csharp/reference/README.md

```diff
@@ -62,6 +67,9 @@ While C# is primarily an OO language, lots of functional concepts have been adde
   - Collections: reducing
   - Iterators (yield)
   - [Generics][generics]
+    - Basic
```

Comment:
This covers the basics of generics I assume? If so, I don't think it really is a separate Concept, but is covered by its parent "Generics" concept.